src/main/java/net/openhft/chronicle/bytes/internal/BytesInternal.java

```diff
@@ -3089,7 +3089,8 @@ public static void appendTimeMillis(@NotNull ByteStringAppender b, long timeInMS
     public static boolean equalBytesAny(@NotNull BytesStore b1, @NotNull BytesStore b2, @NonNegative long readRemaining)
             throws BufferUnderflowException, IllegalStateException {
 
-        if (b1.readRemaining() < readRemaining)
+        if (b1.readRemaining() < readRemaining
```

Comment:
could use Math.min() .. but this is ok as well

Reply:
Done